### ISSUE TITLE
"Queue" build documentation doesn't specify the format for the "parameters" parameter

### ISSUE THREAD

**Issue by jevansaks** [MicrosoftDocs/vsts-rest-api-specs#188]
It just says it's type string and it's the "parameters" for a build. I had to find from other sources on the internet that people figured out that it's a Json-serialized string. So it has to be:

```json
    "parameters" : "{ 'x': '1', 'y': '2' }"
```

The specific page with the issue is https://docs.microsoft.com/en-us/rest/api/azure/devops/build/builds/queue?view=azure-devops-rest-5.0 but I can't file a bug on that page.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d4fa6f24-f644-28c0-07d7-112cb9d401f4
* Version Independent ID: 46013477-6ce4-6d8a-cdfe-55c55db3e518
* Content: [Builds (Azure DevOps Build)](https://docs.microsoft.com/en-us/rest/api/azure/devops/build/builds?view=azure-devops-rest-5.0)
* Content Source: [docs-ref-autogen/5.0/build/Builds.yml](https://github.com/MicrosoftDocs/vsts-docs-rest-apis/blob/live/docs-ref-autogen/5.0/build/Builds.yml)
* Product: **devops**
* Technology: **devops-ref**
* GitHub Login: @wnjenkin
* Microsoft Alias: **whjenkin**

**Comment by frackham** [comment]
Yes, particularly as release pipelines both the format and the name of the property for supplying run-time parameters/variables is different. This is a really quick win and source of confusion.  Compare: https://docs.microsoft.com/en-us/rest/api/azure/devops/build/builds/queue?view=azure-devops-rest-5.0 and https://docs.microsoft.com/en-us/rest/api/azure/devops/release/releases/create?view=azure-devops-rest-5.0 .

Also an example would help.

**Comment by jeffpatton1971** [comment]
Agreed, documentation seems to be lacking in a lot of respects, as well as at least some minimally functional examples.

**Comment by cveld** [comment]
How to provide template parameter values in the build queue rest api payload? 

Also see StackOverflow item https://stackoverflow.com/questions/60852825/azure-devops-yaml-pipeline-parameters-not-working-from-rest-api-trigger#

**Comment by xenibyte** [comment]
To make things even more confusing, after testing it, it looks like those `parameters` for a build pipeline are actually for setting runtime `variables` and not setting the runtime `parameters`. Personally it's actually more convenient to have it as variables but ideally it should be well defined as variables and have the option to also set the runtime parameters for when that's needed.
